### PR TITLE
Add new header font

### PR DESCRIPTION
**Pull request by luacmartins**
### Details
Updates our header font to New Kansas

### Fixed Issues
$ https://github.com/Expensify/App/issues/13686

### Tests
Verify that the following places are using the new header font

1. The `Chats`text in the top left of the LHN
2. The amount in `+ > Send money` 
3. The card headers in the Workspace subpages, e.g. `Workspace > Issue cards`
4. The display name in `Settings`, `Chat header > User details` and `Workspace settings page`
5. The `Not found` page. Type an address that doesn't exist in the browser, e.g. `/test`. Can only be tested on Web and mWeb.
6. The success page in `Security > Change password`. You'll have to change your password to see this page.
7. The success page in `Concierge chat > Request a call`. Enter your details in the modal and request a call.
8. The success page in `Payments > Add bank account`. Connect with Plaid, selecting `Citi` and the sandbox credentials `user_good`, `pass_good`.

C+ and QA can skip the following steps

10. The `Activate wallet` success page. Follow [this SO](https://stackoverflowteams.com/c/expensify/questions/10607/11542#11542) to test the flow and get to the success page.
11. On another account, run `./clitools.sh validator:wallet` and then send money to your test account `+ > Send money` and pay with Expensify Wallet.
12. On your test account, navigate to `Payments > Transfer balance`. Verify that the success page has the new font.

- [x] Verify that no errors appear in the JS console

### Offline tests
N/A

### QA Steps
Same as test steps.

- [ ] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

**BNP**

web

**Workspace cards**
web

**Display names**
web-details
web-profile
web-workspace

**LHN header**
web

**Not found page**
web

**Activate wallet**
web

**Add bank account**
web

**Password confirm**
web

**Request call**
web

**Transfer balance**
web

**Room names**
web

**Protected PDF**
web-pdf

**Drag and drop modal**
web-drag

**Terms Step**
web

</details>

<details>
<summary>Mobile Web - Chrome</summary>

**BNP**
chrome

**Workspace cards**
chrome

**Display names**
chrome-details
chrome-profile
chrome-workspace

**LHN header**
chrome

**Not found page**
chrome

**Add bank account**
chrome

**Password confirm**
chrome

**Request call**
chrome

**Transfer balance**
chrome

**Room names**
chrome

</details>

<details>
<summary>Mobile Web - Safari</summary>

**BNP**
safari

**Workspace cards**
safari

**Display names**
safari-details
safari-profile
safari-workspace

**LHN header**
safari

**Not found page**
safari

**Add bank account**
safari

**Password confirm**
safari

**Request call**
safari

**Transfer balance**
safari

**Room names**
safari

</details>

<details>
<summary>Desktop</summary>

**BNP**
desktop

**Workspace cards**
desktop

**Display names**
desktop-details
desktop-profile
desktop-workspace

**LHN header**
desktop

**Add bank account**
desktop

**Password confirm**
desktop

**Request call**
desktop

**Transfer balance**
desktop

**Room names**
desktop

**Protected PDF**
desktop-pdf

**Drag and drop modal**
desktop-drag

</details>

<details>
<summary>iOS</summary>

**BNP**
ios

**Workspace cards**
ios

**Display names**
ios-details
ios-profile
ios-workspace

**LHN header**
ios

**Add bank account**
ios

**Password confirm**
ios

**Request call**
ios

**Transfer balance**
ios

**Room names**
ios

</details>

<details>
<summary>Android</summary>

**BNP**
android

**Workspace cards**
android

**Display names**
android-details
android-profile
android-workspace

**LHN header**
android

**Add bank account**
android

**Password confirm**
android

**Request call**
android

**Transfer balance**
android

**Room names**
android

</details>

### CODE REVIEW

File: src/pages/settings/Payments/TransferBalancePage.js
Comment (luacmartins) [inline src/pages/settings/Payments/TransferBalancePage.js:211]: I saw the console error below while testing and this style change fixes it.
error

**Comment by luacmartins** [comment]
This is working well for me locally. Ready for review!

**Comment by luacmartins** [comment]
@shawnborton adding you as a reviewer!

File: assets/css/fonts.css
Comment (Luke9389) [inline assets/css/fonts.css:55]: I think we can actually use the same value for font-family here. 
Then when we want to use the italic version we can put `font-style: italic` in the style object that's being applied to the element. LMK if that makes sense, this is something I just figured out today on [this PR](https://github.com/Expensify/App/pull/12457/files). You can see an example there with bold fonts. 

In general, I think this means we should refactor the fonts to follow this technique. If you don't want to do it here, we can go with your current code, and then I'll open a PR later to clean up all the fonts at once.

Comment (luacmartins) [reply assets/css/fonts.css:55]: Good to know 😄 I can update `New Kansas` to use the same font-family since we are introducing it here, but I think it's best to clean up the other fonts in a separate PR to separate the concerns and chances for regressions.

Comment (Luke9389) [reply assets/css/fonts.css]: I agree. I'll make that follow up PR once this one is merge. Thanks @luacmartins

Comment (luacmartins) [reply assets/css/fonts.css]: Thank you!

File: assets/css/fonts.css
Comment (shawnborton) [inline assets/css/fonts.css:44]: The font-family here should just be `ExpensifyNewKansas`. Medium, or Medium Italic, are just weights/styles of the font family.

File: assets/css/fonts.css
Comment (shawnborton) [inline assets/css/fonts.css:51]: Similar comment as above, this should just say `ExpensifyNewKansas`

File: src/styles/styles.js
Comment (shawnborton) [inline src/styles/styles.js:285]: I wonder if something more generic like `headlineFont` makes more sense here?

File: src/styles/styles.js
Comment (shawnborton) [inline src/styles/styles.js:2013]: Rather than needing to explicitly call in `fontFamily.EXP_NEW_KANSAS_MEDIUM` - is there a way we can just add something similar to a Sass mixin that would basically work like a variable and pull in the fontFamily and fontWeight for us? 

Something in the Sass world would be like `@extend .headlineFont`

Basically this way we only need to maintain our headline style (font family and weight) in one place.

**Comment by shawnborton** [comment]
Looking good! Left some comments for ya, can't wait for this one to be merged 😻

File: src/pages/settings/Payments/TransferBalancePage.js
Comment (luacmartins) [reply src/pages/settings/Payments/TransferBalancePage.js:211]: I didn't notice any UI changes during my tests. Did I miss something?

File: assets/css/fonts.css
Comment (luacmartins) [reply assets/css/fonts.css]: We can get close to this! iOS uses `Expensify New Kansas` (with spaces) and I'm not sure if we can change this since it's the name stored in the font file.

For android, we'll have to switch to [XML fonts](https://github.com/jsamr/react-native-font-demo#android) and specify each combo of font-weight/font-style. Otherwise, android uses the file names and using a different value as font-family would break the font.

I pushed the changes to make this happen. Let me know if you still have any concerns!

File: src/styles/styles.js
Comment (luacmartins) [reply src/styles/styles.js]: Are you thinking of something like:

```
const headlineFont = {
   fontFamily: fontFamily.EXP_NEW_KANSAS,
   fontWeight: '500'
}
```

and then call `...headlineFont` in each usage, e.g.:

```
    displayName: {
        ...headlineFont,
        fontSize: variables.fontSizeLarge,
        color: themeColors.heading,
    },
```

Would that solve your concern?

**Comment by luacmartins** [comment]
Merged main and implemented the suggestions. Ready for another round!

File: src/styles/styles.js
Comment (luacmartins) [reply src/styles/styles.js]: I pushed the changes above. Let me know if that's not what you had in mind and we can update it again!

**Comment by luacmartins** [comment]
Note to self: keep track of [this PR](https://github.com/Expensify/App/pull/13606#issuecomment-1361329474) and update the font in the [DeepLinkWrapper page](https://github.com/Expensify/App/pull/13606/files#diff-8db92687aa1f826d3c976cf4638b9aa7a5c5316c1bb1dddc22d1a2e473aab751R1) if that PR is merged before this one.

**Comment by luacmartins** [comment]
@shawnborton do we want to update this generic error page too? 

Screenshot 2022-12-21 at 2 02 03 PM

File: android/app/src/main/java/com/expensify/chat/MainApplication.java
Comment (roryabraham) [inline android/app/src/main/java/com/expensify/chat/MainApplication.java:70]: @luacmartins Why did you add the font in this way? This would be the only xml font in the application. I think we should really stick with the pattern established for other native fonts here:

- put a native `.otf` font in `/assets/fonts/native`
- Run `npx react-native-asset – that will:
    - Add the native font to `android/link-assets-manifest.json`
    - Put a duplicate font in `android/app/src/main/assets/fonts` instead of in `android/res/font`

**Review by roryabraham — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/com/expensify/chat/MainApplication.java
Comment (roryabraham) [reply android/app/src/main/java/com/expensify/chat/MainApplication.java:70]: Then no native code should be required to load the font

**Review by roryabraham — DISMISSED** [review]
Also, since we removed the last `.otf` font from `assets/fonts/web`, can we also update the webpack config [here](https://github.com/Expensify/App/blob/8b14312cf491d8c191a7bb8f8640b3e4e0513d92/config/webpack/webpack.common.js#L192) so that it only loads `.woff2` and `.woff`?

File: android/app/src/main/java/com/expensify/chat/MainApplication.java
Comment (luacmartins) [reply android/app/src/main/java/com/expensify/chat/MainApplication.java:70]: I explained a bit of it [here](https://github.com/Expensify/App/pull/13714#discussion_r1054817989). Is there another way to add it and be able to use `Expensify New Kansas` as `fontFamily` across all platforms?

Based on [this conversation](https://github.com/Expensify/App/pull/13714#discussion_r1053792219), it seems like @Luke9389 will work on a follow up PR to rename the other fonts to use the correct fontFamily name as well. So we'd solve any inconsistencies there.

Comment (roryabraham) [reply android/app/src/main/java/com/expensify/chat/MainApplication.java]: As discussed 1:1, let's revert that change from this PR and then create a follow-up issue to address the XML fontFamily thingy. I'd love to work on a way to make that improvement upstream without compromising the ease of font asset management we get from https://github.com/unimonkiez/react-native-asset

File: assets/css/fonts.css
Comment (luacmartins) [reply assets/css/fonts.css:44]: @shawnborton @roryabraham and I discussed this and we agreed that it's best to tackle all the font-family grouping in a follow-up PR for a couple of reasons:

1. If we do it here, it would introduce yet another way of importing fonts, which touches native android code - we don't want that
2. We can implement a better solution straight in the library we use to link assets, making it easier to import fonts in the future

Comment (luacmartins) [reply assets/css/fonts.css:44]: I'll keep these font-families as is in favor of [this issue](https://github.com/Expensify/App/issues/13772)

**Comment by luacmartins** [comment]
Ok, reverted the XML fonts change and remove `otf` fonts from webpack config. Ready for another round of reviews!

**Comment by shawnborton** [comment]
> do we want to update this generic error page too?

@luacmartins Yes! Let's at least update the headline there to use Expensify New Kansas. We can follow up and create a separate issue to add a nicer illustration. What is the easiest way to access that page btw?

File: assets/css/fonts.css
Comment (shawnborton) [reply assets/css/fonts.css:44]: I don't think I am following, it's really just a matter of changing the names around no?

**Comment by shawnborton** [comment]
For this screen here:
image


We should also update the currency symbol to use New Kansas.

File: assets/css/fonts.css
Comment (luacmartins) [reply assets/css/fonts.css:44]: Unfortunately not. Doing so breaks the font on android as android uses the font filename, i.e. ExpensifyNewKansas-Medium, etc as `fontFamily` and renaming the files would break iOS since iOS needs the filename to match the font's postscript name.

As I explained above, we can get around this by using [XML fonts](https://github.com/jsamr/react-native-font-demo#android) for android and that's what the issue I linked above will do!

Comment (shawnborton) [reply assets/css/fonts.css:44]: Got it got it, thanks for clarifying! Okay, then yeah let's get this implemented and you can follow up and fix all of the fonts :)

**Comment by luacmartins** [comment]
>@luacmartins Yes! Let's at least update the headline there to use Expensify New Kansas. We can follow up and create a separate issue to add a nicer illustration. What is the easiest way to access that page btw?

@shawnborton I'm not sure if there's an easy way on staging, since that means we'd have an error in the App codebase and our users should (hopefully) never see that page. On dev, you can comment out [this if statement](https://github.com/Expensify/App/blob/45b69192125b1741acb9d24a8b6d7a43f069afc6/src/components/ErrorBoundary/BaseErrorBoundary.js#L50-L52) to always render that page.

BTW this is how the page looks like with the new font:

<details>
<summary>Web</summary>
web

</details>

<details>
<summary>Desktop</summary>
desktop

</details>

<details>
<summary>Safari</summary>
safari

</details>

<details>
<summary>iOS</summary>
ios

</details>

<details>
<summary>Chrome</summary>

chrome

</details>

<details>
<summary>Android</summary>
android

</details>

**Comment by shawnborton** [comment]
Nice, looks good!

**Comment by shawnborton** [comment]
Testing now, gonna report a few things as I see them

For the Chats header, could we bump up the font size to something like 24px? Maybe we can reuse an existing font size var if something is close
image

**Comment by luacmartins** [comment]
Cool, I'll address your comments once you are done testing!

**Comment by shawnborton** [comment]
For workspace editor cards, let's bump that font size up as well:
image


So maybe we can have both the workspace editor card titles and the `Chats` header both use 22px? Hoping we can try to standardize on our font sizes a bit so we don't have a million different sizes, but we can do some of that as a second pass in a separate PR as well.

**Comment by shawnborton** [comment]
While we're at it, let's make the display name in Profile View use 22px as well:
image

**Comment by shawnborton** [comment]
Same with this one, let's do 22px:
image

**Comment by shawnborton** [comment]
Maybe we should have Wallet Balance match the same style as the Send/Request big number pad as well:
image

Same on the transfer page too:
image

**Comment by shawnborton** [comment]
Okay, I think that concludes my testing for now 🤣

**Comment by luacmartins** [comment]
Updated with the latest suggestions! I triggered another testing build. I'll update the screenshots when we are done reviewing!

**Comment by shawnborton** [comment]
Hmm that latest link looks like the old version I tested - any idea what's up?

**Comment by luacmartins** [comment]
The links seem correct to me, but I had to disable/clear my cache and refresh to see the new changes. Could you try that?

**Comment by shawnborton** [comment]
Ah yes, much better!!
image

**Review by parasharrajat — CHANGES_REQUESTED** [review]
BUG: The report page details font is incorrect. 
Screenshot 2022-12-27 20:32:34

**Comment by shawnborton** [comment]
@parasharrajat The answer is no to each of your questions [here](https://github.com/Expensify/App/pull/13714#issuecomment-1365969052), those screenshots look fine as they are.

**Review by parasharrajat — CHANGES_REQUESTED** [review]

**Review by parasharrajat — CHANGES_REQUESTED** [review]

**Comment by shawnborton** [comment]
> What about the Drop UI header?
> What about password protected PDF?

Good catches on those, yeah I think it makes sense to update them to match the other empty state headings cc @luacmartins

**Comment by luacmartins** [comment]
I'm ooo until Jan 4. I'll come back to this then! Thanks for the comments so far!

File: assets/css/fonts.css
Comment (Luke9389) [reply assets/css/fonts.css:44]: This was a big gotcha for me on the fonts PR too.

File: src/pages/settings/Payments/TransferBalancePage.js
Comment (luacmartins) [reply src/pages/settings/Payments/TransferBalancePage.js]: Done! Thanks for clarifying!

File: android/link-assets-manifest.json
Comment (luacmartins) [reply android/link-assets-manifest.json:30]: I updated the font files with the new ones provided by Shawn [here](https://github.com/Expensify/App/issues/13686#issue-1502984381) and then ran `npx react-native-asset` to link them. These changes were made by the script.

File: android/link-assets-manifest.json
Comment (luacmartins) [reply android/link-assets-manifest.json:33]: Could you clarify what you mean by that?

**Comment by luacmartins** [comment]
@shawnborton should we update these bold values to the new font as well?
Screenshot 2023-01-04 at 2 20 17 PM

**Comment by luacmartins** [comment]
I believe that I addressed all comments. This is ready for another round of reviews!

**Comment by shawnborton** [comment]
@luacmartins I suppose we can use the headline font in the terms page, it can't hurt!

**Comment by luacmartins** [comment]
Update fonts in TermsStep

Screenshot 2023-01-05 at 9 21 24 AM

**Comment by shawnborton** [comment]
Looks great. That terms page could use an updated border color. All good if you don't want to add it into this PR, but ideally we could just use the default border color from our styles for this.

**Comment by luacmartins** [comment]
👍 Update the border color too:

Screenshot 2023-01-05 at 9 28 20 AM

**Comment by shawnborton** [comment]
Amazing, looks great to me!

**Comment by luacmartins** [comment]
@MariaHCD @Luke9389 @shawnborton @roryabraham @parasharrajat this is ready for review!

**Review by Luke9389 — DISMISSED** [review]
Looks fantastic, thanks!

**Review by MariaHCD — DISMISSED** [review]
LGTM ✨

**Review by shawnborton — DISMISSED** [review]

**Comment by shawnborton** [comment]
Looks good to me too. I will be OOO for the rest of the day so please feel free to merge without me, thanks!

**Review by parasharrajat — APPROVED** [review]
~Reviewing now.~

Please wait for the final testing. I accidentally approved it.

**Review by parasharrajat — CHANGES_REQUESTED** [review]
Wait for the final test. Waiting on my IP to be approved.

File: src/components/Header.js
Comment (luacmartins) [reply src/components/Header.js]: Sure, that works too. Updated!

File: src/pages/ReportDetailsPage.js
Comment (luacmartins) [reply src/pages/ReportDetailsPage.js:143]: `reportDetailsTitleContainer` sets a fixed height of 170px to the parent container and increasing the font size in `displayNames` caused the subtitle to be clipped. I reduced it to `mb2` to be consistent with the margin used in `displayNames` and get rid of the clipping.

**Comment by luacmartins** [comment]
We are looking good. Just missing the completed checklist and another approval.

**Review by parasharrajat — APPROVED** [review]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

:ribbon: :eyes: :ribbon:  C+ reviewed

**Comment by luacmartins** [comment]
@parasharrajat thanks and I assigned you to the issue. We are just missing one internal approval now. @Luke9389 would you mind reviewing again since the other reviewers are out for the weekend already?

**Review by Luke9389 — APPROVED** [review]

**Comment by luacmartins** [comment]
Not sure why this was marked as a blocker. The TTI was actually reduced by 13%!

**Comment by roryabraham** [comment]
> Not sure why this was marked as a blocker. The TTI was actually reduced by 13%!

Made a follow-up issue to investigate this: https://github.com/Expensify/App/issues/14123

**Pull request merged**
